### ISSUE TITLE
Add link to source

### ISSUE THREAD

**Issue by ivmartel** [ankitskvmdam/clean-jsdoc-theme#71]
**Is your feature request related to a problem? Please describe.**

It would be nice to have a link to the source code from all documented elements, as in this [example](https://cancerberosgx.github.io/jsdoc-templates-demo/demo/docdash/Apple.html) (see the constructor).

**Describe the solution you'd like**

Add an option to include or not source links.

**Describe alternatives you've considered**

None.

**Additional context**

None.

**Comment by ankit-kumar-kr** [comment]
I think this feature is already there. Visit:  https://github.com/ankitskvmdam/clean-jsdoc-theme/issues/60#issuecomment-807074900

**Comment by ivmartel** [comment]
I do not have links to source below functions / methods documentation even with the `outputSourceFiles` options set to true (its default, see [ref](https://jsdoc.app/about-configuring-default-template.html#generating-pretty-printed-source-files)). 

Here is my config
```
{
    "plugins": ["node_modules/jsdoc/plugins/markdown"],
    "markdown": {
      "idInHeadings": true
    },
    "source": {
      "include": [
        "src",
        "tests"
      ]
    },
    "templates": {
      "default": {
        "outputSourceFiles": true,
        "staticFiles": {
          "include": ["resources/doc/img"]
        }
      }
    },
    "opts": {
      "recurse": true,
      "readme": "resources/doc/readme-doc.md",
      "destination": "build/doc",
      "template": "node_modules/clean-jsdoc-theme",
      "tutorials": "resources/doc/tutorials",
      "package": "package.json",
      "theme_opts": {
        "theme": "dark"
      }
    }
}

```

**Comment by ankitskvmdam** [comment]
Oh.. Got it .. There is some bug. Need to look into it.

**Comment by ankitskvmdam** [comment]
Thanks, for reporting this.. I really appreciate your effort. Thank you. This is also fixed in v3.2.7, I will release this version in the coming weekend.
